packages/playground-preview-worker/src/user.do.ts

```diff
@@ -158,17 +160,33 @@ export class UserSession {
 			}
 			return workerResponse;
 		}
+
 		const userSession = this.state.id.toString();
-		const worker = await request.formData();
 
-		const m = worker.get("metadata");
+		let worker: FormData;
+		try {
+			worker = await request.formData();
+		} catch (e) {
+			throw new BadUpload(`Expected valid form data`);
```

Comment:
Is it worth appending the info from the original `e` to this new exception?
In case that would be helpful to the user debugging their problem?